Cargo.toml

```diff
@@ -48,7 +49,7 @@ version = "0.6"
 [features]
 nightly = ["radix_51"]
 default = ["std"]
-std = ["rand"]
+std = ["rand", "subtle/std"]
```

Comment:
I don't think you're actually using any of the std-dependent features of `subtle`? I enabled them just in case you want to, though.